include/datafile/UpgradeTo13ExtendedNoteRange.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * UpgradeTo13ExtendedNoteRange.h
+ *   Upgrades the extended note range
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#pragma once
+
+#ifndef LMMS_UPGRADEEXTENDEDNOTERANGE_H
+#define LMMS_UPGRADEEXTENDEDNOTERANGE_H
```

Comment:
I know that this was the original header guard, but I still think it should be formatted differently:
```suggestion
#ifndef LMMS_UPGRADE_EXTENDED_NOTE_RANGE_H
#define LMMS_UPGRADE_EXTENDED_NOTE_RANGE_H
```